templates/readme.md

```diff
@@ -0,0 +1,29 @@
+# /* {tagName} */
+An HTML custom element implementing the `</* {tagName} */>` tag.
+
+[ Add a nice image here ]
+
+## Setup
+
+### Installation
+```
+npm i /* {tagName} */
+```
+
+---
+
+```Html
+<script src="node_modules//* {tagName} *///* {tagName} */.js"></script>
+```
+or if you're bundling
+```Javascript
+import "/* {tagName} */";
+// or
+require("/* {tagName} */");
```

Comment:
Our code doesn't support CommonJS module exports afaik? Which I guess isn't a problem with custom elements unless someone wants to extend our custom element I guess.

Reply:
Yea, this is more for people who will be using a bundler. As is our code is fine not supporting CommonJS, more modular that way.